### PR TITLE
docs(update): task worker metrics config

### PR DESCRIPTION
**Pull request by danielkelemen**
* document the new task worker metrics behaviour and config

related to CAM-13640

### CODE REVIEW

File: content/update/minor/716-to-717/jboss.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+The following steps are required to update the Camunda web applications Cockpit, Tasklist, and Admin on a JBoss/Wildfly instance:
+
+1. Undeploy an existing web application with a name like `camunda-webapp`
+2. Download the Camunda web application archive from our [Maven Nexus Server][webapp-jboss].
+   Alternatively, switch to the private repository for the enterprise version (credentials from license required).
+   Choose the correct version named `$PLATFORM_VERSION/camunda-webapp-jboss.war`.
+3. Deploy the web application archive to your JBoss/Wildfly instance.
+
+
+[update-guide]: {{< ref "/update/minor/715-to-716/_index.md" >}}
```
Comment (tasso94) [inline content/update/minor/716-to-717/jboss.md:120]:
```suggestion
[update-guide]: {{< ref "/update/minor/716-to-717/_index.md" >}}
```

File: content/update/minor/716-to-717/tomcat.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+## Update Cockpit, Tasklist, and Admin
+
+The following steps are required to update the Camunda web applications Cockpit, Tasklist, and Admin on a Tomcat instance:
+
+1. Undeploy an existing web application with a name like `camunda-webapp`
+2. Download the Camunda web application archive from our [Maven Nexus Server][nexus-webapp]. Alternatively, switch to the private repository for the enterprise version (credentials from license required). Choose the correct version named `$PLATFORM_VERSION/camunda-webapp-tomcat-$PLATFORM_VERSION.war`.
+3. Deploy the web application archive to your Tomcat instance.
+
+[update-guide]: {{< ref "/update/minor/715-to-716/_index.md" >}}
```
Comment (tasso94) [inline content/update/minor/716-to-717/tomcat.md:101]:
```suggestion
[update-guide]: {{< ref "/update/minor/716-to-717/_index.md" >}}
```

File: content/update/minor/716-to-717/was.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+## Cockpit, Tasklist, and Admin
+
+The following steps are required to update the Camunda web applications Cockpit, Tasklist, and Admin on an IBM WebSphere instance:
+
+1. Deploy the web application `$WAS_DISTRIBUTION/webapps/camunda-webapp-ee-was-$PLATFORM_VERSION.war` to your IBM WebSphere instance.
+2. Associate the web application with the `Camunda` shared library.
+
+[configuration-location]: {{< ref "/reference/deployment-descriptors/descriptors/bpm-platform-xml.md" >}}
+[update-guide]: {{< ref "/update/minor/715-to-716/_index.md" >}}
```
Comment (tasso94) [inline content/update/minor/716-to-717/was.md:114]:
```suggestion
[update-guide]: {{< ref "/update/minor/716-to-717/_index.md" >}}
```

File: content/update/minor/716-to-717/wls.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+## REST API
+
+Deploy the web application `$WLS_DISTRIBUTION/webapps/camunda-engine-rest-$PLATFORM_VERSION-wls.war` to your Oracle WebLogic instance.
+
+## Cockpit, Tasklist, and Admin
+
+Deploy the web application `$WLS_DISTRIBUTION/webapps/camunda-webapp-ee-wls-$PLATFORM_VERSION.war` to your Oracle WebLogic instance.
+
+[configuration-location]: {{< ref "/reference/deployment-descriptors/descriptors/bpm-platform-xml.md" >}}
+[update-guide]: {{< ref "/update/minor/715-to-716/_index.md" >}}
```
Comment (tasso94) [inline content/update/minor/716-to-717/wls.md:110]:
```suggestion
[update-guide]: {{< ref "/update/minor/716-to-717/_index.md" >}}
```

File: content/update/minor/716-to-717/_index.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+# Standalone Web Application
+
+If you use a standalone web application, replace the current `war` artifact by its new version. 
+Take the following steps to complete the update:
+
+1. Undeploy the current version of the standalone web application.
+2. Update the database to the new schema as described in the [database update](#database-updates) section.
+3. Configure the database as described in the [installation]({{< ref "/installation/standalone-webapplication.md#database-configuration" >}})
+   section. **Note** that with 7.16 the standalone web applications use **HikariCP** for data sources instead of Apache Commons DBCP. Replace the
+   `targetDataSource`'s bean class to `com.zaxxer.hikari.HikariDataSource` and rename the `url` parameter of the data source to `jdbcUrl`.
```
Comment (tasso94) [inline content/update/minor/716-to-717/_index.md:69]:
```suggestion
   section.
```

File: content/update/minor/716-to-717/_index.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+4. Deploy the new and configured standalone web application to the server.
+
+# Set Variables on Process Instance Migration
+
+We have extended the [Process Instance Migration Batch Operation]({{< ref "/user-guide/process-engine/process-instance-migration.md#set-variables-to-process-instances" >}}) to set variables into the process instances' scope.
+Please bear in mind that the [usage of new features]({{< ref "/rolling-update.md#usage-of-new-features" >}}) during a rolling update 
+leads to unexpected behavior and therefore must be avoided: When a migration batch with variables is 
+created during a rolling update, variables might or might not be set depending on the executing engine (old/new engine), 
+and batch variables might not be removed when an old engine executes the monitor job.
```
Comment (tasso94) [inline content/update/minor/716-to-717/_index.md:78]:
```suggestion
```

File: content/update/minor/716-to-717/_index.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+    pre: "Update from `7.16.x` to `7.17.0`."
+
+---
+
+This document guides you through the update from Camunda Platform `7.16.x` to `7.17.0`. It covers these use cases:
+
+1. For administrators and developers: [Database Updates](#database-updates)
+1. For administrators and developers: [Full Distribution Update](#full-distribution)
+1. For administrators: [Standalone Web Application](#standalone-web-application)
+1. For administrators: [Set Variables on Process Instance Migration](#set-variables-on-process-instance-migration)
```
Comment (tasso94) [inline content/update/minor/716-to-717/_index.md:21]:
```suggestion
1. For administrators: [Task Worker Metrics](#task-worker-metrics)
```

File: content/update/minor/716-to-717/_index.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+We have extended the [Process Instance Migration Batch Operation]({{< ref "/user-guide/process-engine/process-instance-migration.md#set-variables-to-process-instances" >}}) to set variables into the process instances' scope.
+Please bear in mind that the [usage of new features]({{< ref "/rolling-update.md#usage-of-new-features" >}}) during a rolling update 
+leads to unexpected behavior and therefore must be avoided: When a migration batch with variables is 
+created during a rolling update, variables might or might not be set depending on the executing engine (old/new engine), 
+and batch variables might not be removed when an old engine executes the monitor job.
+
+# Task Worker Metrics
+
+Starting from version 7.17 the task worker metrics are displayed by default.
```
Comment (tasso94) [inline content/update/minor/716-to-717/_index.md:82]:
```suggestion
Starting from version 7.17, the task worker metrics are displayed by default.
```

File: content/update/minor/716-to-717/_index.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+If this causes slow page loading you can turn it off in the  [admin webapp configuration.]({{< ref "/webapps/admin/configuration.md#task-worker-metrics" >}})
```
Comment (tasso94) [inline content/update/minor/716-to-717/_index.md:83]:
```suggestion
If this causes slow page loading, you can turn it off in the  [admin webapp configuration.]({{< ref "/webapps/admin/configuration.md#task-worker-metrics" >}})
```

File: content/update/minor/716-to-717/jboss.md [hunk 10]
```diff
@@ -0,0 +1,122 @@
+---
+
+title: "Update a JBoss/Wildfly Installation from 7.16 to 7.17"
```
Comment (tasso94) [inline content/update/minor/716-to-717/jboss.md:3]:
```suggestion
title: "Update a JBoss EAP 6 or Wildfly / JBoss EAP 7 Installation from 7.16 to 7.17"
```

File: content/update/minor/716-to-717/jboss.md [hunk 11]
```diff
@@ -0,0 +1,122 @@
...
+
+menu:
+  main:
+    name: "JBoss AS/Wildfly"
```
Comment (tasso94) [inline content/update/minor/716-to-717/jboss.md:7]:
```suggestion
    name: "JBoss EAP 6 or Wildfly / JBoss EAP 7"
```

File: content/update/minor/716-to-717/jboss.md [hunk 12]
```diff
@@ -0,0 +1,122 @@
...
+    identifier: "migration-guide-717-jboss"
+    parent: "migration-guide-717"
+
+---
+
+The following steps describe how to update the Camunda artifacts on a JBoss AS
+7 and Wildfly server in a shared process engine scenario. For the entire
```
Comment (tasso94) [inline content/update/minor/716-to-717/jboss.md:14]:
```suggestion
The following steps describe how to update the Camunda artifacts on a JBoss EAP 6 or Wildfly / JBoss EAP 7 in a shared process engine scenario. For the entire
```

File: content/update/minor/716-to-717/_index.md [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+1. Update the Camunda libraries and applications inside the application server
+2. Migrate custom process applications
+
+Before starting, make sure that you have downloaded the Camunda Platform 7.17 distribution for the application server you use. It contains the SQL scripts and libraries required for the update. This guide assumes you have unpacked the distribution to a path named `$DISTRIBUTION_PATH`.
+
+## Camunda Libraries and Applications
+
+Please choose the application server you are working with from the following list:
+
+* [JBoss AS/Wildfly]({{< ref "/update/minor/716-to-717/jboss.md" >}})
```
Comment (tasso94) [inline content/update/minor/716-to-717/_index.md:44]:
```suggestion
* [JBoss EAP 6 or Wildfly / JBoss EAP 7]({{< ref "/update/minor/716-to-717/jboss.md" >}})
```

**Review by tasso94 — CHANGES_REQUESTED** [review]
Looks good in general. 🙂 👍 

Once the technical review is done, let's assign `@christinaausley` for review as well.

I added some review hints below:

**Review by tasso94 — APPROVED** [review]
Looks good to me. Let's request another review from @christinaausley.

**Review by christinaausley — APPROVED** [review]
Added a commit with my editorial/technical revisions -- looks great!

**Pull request merged**